internal/runner/send_register.go

```
@@ -8,48 +8,39 @@ import (
+		Registered:       true,
+		LastHeartbeat:    time.Now(),
+		Version:          version,
+		Mode:             cfg.Mode,
+		Plugins:          plugins,
+		Actions:          actions,
+		PayloadEndpoints: payloadEndpoints,
 	}
-	register.PayloadEndpoints = json.RawMessage(payloadInjectorsJSON)
 
 	payloadBuf := new(bytes.Buffer)
 	json.NewEncoder(payloadBuf).Encode(register)
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)